py/client-ticking/src/pydeephaven_ticking/_util.py

```diff
@@ -3,32 +3,33 @@
 #
 from __future__ import annotations
 
-from typing import List, Sequence, TypeVar, Union
-import pydeephaven_ticking._core as dhc
+from typing import Any, Optional, Sequence, Union
 
-T = TypeVar("T")
+import pydeephaven_ticking._core as dhc
 
 
-def _to_sequence(v: Union[T, Sequence[T]] = None) -> Sequence[T]:
+def _to_sequence(v: Optional[Any] = None) -> Sequence[Any]:
```

Comment:
Are you sure you want this change?  It looks like you are losing type info in a negative way.